database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -16,9 +16,12 @@ public function up()
         Schema::create('users', function (Blueprint $table) {
             $table->id();
             $table->string('name');
+            $table->string('first_name');
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->timestamp('email_verified_at')->nullable();
```

Comment:
why not just a 0/1 value if the email was verified? 
Usually you don't care when a user validated his account